src/global.store.ts

```diff
@@ -334,6 +336,48 @@ export class GlobalStore implements IGlobalStore {
         this._actionLogger.SetLogger(logger);
     }
 
+    /**
+     * Summary: Expose an API collection from a Partner-level that other partners can later consume. This allows partners to derive data without forcing partners to know the state structure.
+     * 
+     * @access public
+     * 
+     * @param {string} source Name of the application exposing an derived state API
+     * @param {Record<string, any>} api The collection of APIs of derived state selectors.
+     * @param {boolean} mergeApi If the source application already exposed an API set, merge the new API being passed in.
+     * 
+     */
+    ExposeDerivedState(source: string, api: Record<string, any>, mergeApi = false) {
```

Comment:
Rename suggestions
`ExposeDerivedState` --> `AddSelector`
`api` -->`selector`
`mergeApi` -> `mergeSelector`

Reply:
100% these make far more sense as names. I opted to keep them as plural i.e. (`AddSelectors`, `selectors`, `mergeSelectors`) let me know your thoughts :)